data/serializer/src/main/java/tech/pegasys/teku/api/schema/capella/BeaconStateCapella.java

```diff
@@ -165,5 +182,10 @@ public static void applyCapellaFields(
 
     state.setNextWithdrawalIndex(instance.nextWithdrawalIndex);
     state.setNextWithdrawalValidatorIndex(instance.nextWithdrawalValidatorIndex);
+    state.setHistoricalSummaries(
+        historicalSummariesSchema.createFromElements(
+            instance.historicalSummaries.stream()
+                .map(HistoricalSummary::asInternal)
+                .collect(Collectors.toList())));
   }
 }
```

Comment:
Are we missing  `getHistoricalSummariesSchema() { ... }` returning `SszListSchema<HistoricalSummary, ?>`

Reply:
im not sure why schema is like this, i was following the pattern to be honest...

Reply:
My understanding is that the list of historical summaries belongs in the state schema. I did a similar thing for BLSToExecutionChanges in the block body schema. 🤷

Reply:
I think this one I'm ok with leaving alone, as it does seem consistent currently...